src/engraving/dom/cmd.cpp

```diff
@@ -128,6 +129,53 @@ static std::pair<int, int> changedTicksRange(const CmdState& cmdState, const std
     return { startTick, endTick };
 }
 
+//---------------------------------------------------------
+//   resetElementShapePosition
```

Comment:
mismatch between comment and real method name

Reply:
fixed